std/special/compiler_rt.zig

```diff
@@ -471,6 +471,28 @@ test "usesThumb1" {
     //etc.
 }
 
+const use_thumb_1_pre_armv6 = usesThumb1PreArmv6(builtin.arch);
+
+fn usesThumb1PreArmv6(arch: builtin.Arch) bool {
+    return switch (arch) {
+        .thumb => switch (arch.thumb) {
```

Comment:
Slightly more idiomatic:
```suggestion
        .thumb => |sub_arch |switch (sub_arch) {
```